### PR TITLE
FS-4422:deadline-for-applications-welsh-trans-v3

### PR DESCRIPTION
**Pull request by Karunred**
While user testing the withdraw/delete feature, we found applicants were confused about whether a round or window was open and expected to see the submission deadline clearly displayed on their dashboard.

Date format needs should be full month along with welsh translation

### Change description
_A brief description of the pull request_

- [ ] Unit tests and other appropriate tests added or updated
- [ ] README and other documentation has been updated / added (if needed)
- [ ] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
_If manual testing is needed, give suggested testing steps_

### Screenshots of UI changes (if applicable)

image

image

### CODE REVIEW

File: app/filters.py [hunk 1]
```diff
@@ -77,3 +77,18 @@ def kebab_case_to_human(word: str) -> str | None:
...
+
+
+def datetime_format_full_month(value: datetime) -> str:
+    if value:
+        formatted_date = format_datetime(value, format="dd MMMM yyyy ")
+        formatted_date += gettext("at")
+        formatted_date += format_datetime(value, format=" h:mm", rebase=False)
+        formatted_date += format_datetime(value, "a", rebase=False).lower()
+        return formatted_date
+    else:
```
Comment (NarenderRajuB) [inline app/filters.py:93]: we can remove else here

**Review by NarenderRajuB — CHANGES_REQUESTED** [review]
LGTM except this else part

File: app/filters.py (hunk 1 above)
Comment (Karunred) [reply app/filters.py]: @NarenderRajuB, updated as requested and tested accordingly

**Review by NarenderRajuB — APPROVED** [review]
LGTM

**Pull request merged**